extensions/kubeapi/rbac/rbac.go

```diff
@@ -66,3 +72,32 @@ var RoleTemplateGroupVersionResource = schema.GroupVersionResource{
 	Version:  Version,
 	Resource: "roletemplates",
 }
+
+// ProjectRoleTemplateBindingGroupVersionResource is the required Group Version Resource for accessing projectroletemplatebindings in a cluster, using the dynamic client.
+var ProjectRoleTemplateBindingGroupVersionResource = schema.GroupVersionResource{
```

Comment:
Is this diff expected? This is something new for 2.7 right? It seems like 2.8 and 2.9 already have it

Reply:
@anupama2501 I realized this only after reading Ugur's question. Since Public API was introduced in 2.8, we cannot have the following PRTB related methods in 2.7, correct?
- ListProjectRoleTemplateBindings
- GetProjectRoleTemplateBindingsByName

Reply:
ah its going to be a bit tricky. So, for the tests that are using these functions, we would need to update them to use v3 or v1 API specifically for 2.7 and keep 2.8 and 2.9 use public API.

Reply:
Removed public API references here. thank you for calling these out!